transforms/code/license_check/ray/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM docker.io/rayproject/ray:2.9.3-py310
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Copy and install data processing libraries 
+# These are expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=ray:users data-processing-lib-python/ data-processing-lib-python/
```

Comment:
here also copying the library from local, can we make it pip install from package manager

Reply:
it is a convention in the repo